frontend/src/metabase/collections/components/CollectionContent/CollectionContentView.tsx

```diff
@@ -119,31 +88,28 @@ export const CollectionContentView = ({
   };
 
   const handleUploadFile = useCallback<OnFileUpload>(
-    (props: CollectionOrTableIdProps) => {
-      const { collectionId, tableId } = props;
+    (uploadFileArgs: CollectionOrTableIdProps) => {
```

Comment:
The name confused me (thought it was the actual props of this component) so I changed it